R/LearnerDensKDE.R

```diff
@@ -54,10 +54,12 @@ LearnerDensKDE = R6::R6Class("LearnerDensKDE",
         ClassName)))$new()
 
 
-      bw = ifelse(self$param_set$values$bandwidth == "silver",
+      bw = if (isTRUE(self$param_set$values$bandwidth == "silver")) {
```

Comment:
The reverse we want here: if "silver" badwith (the default) we do the complicated calculation, otherwise we keep the value `self$param_set$values$bandwidth`

Reply:
yeah, just saw that